src/containers/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,258 @@
+      <Form
+        noValidate
+        validated={validated}
+        onSubmit={handleSubmit}
+        className="userForm"
+      >
+        <Form.Group controlId="formFirstName">
+          <Form.Label>First Name</Form.Label>
+          <Form.Control
+            type="text"
+            name="firstName"
+            onChange={handleChange}
+            value={formData.firstName}
+            placeholder="Enter first name"
+            required
+          />
+        </Form.Group>
+
+        <Form.Group controlId="formLastName">
+          <Form.Label>Last Name</Form.Label>
+          <Form.Control
+            type="text"
+            name="lastName"
+            onChange={handleChange}
+            value={formData.lastName}
+            placeholder="Enter last name"
+            required
+          />
+        </Form.Group>
+
+        <Form.Group controlId="formEmail">
+          <Form.Label>Email address</Form.Label>
+          <Form.Control
+            type="email"
+            name="email"
+            onChange={handleChange}
+            value={formData.email}
+            placeholder="Enter email"
+            required
+          />
+          <Form.Text className="text-dark">
+            We&apos;ll never share your email with anyone else
+          </Form.Text>
+        </Form.Group>
+
+        <Form.Group controlId="formPassword">
+          <Form.Label>Password</Form.Label>
+          <Form.Control
+            type="password"
+            name="password"
+            onChange={handleChange}
+            value={formData.password}
+            placeholder="Enter password"
+            required
+          />
+          <Form.Text className="text-dark">
+            Password should be at least six characters long
+          </Form.Text>
+        </Form.Group>
+
+        <Form.Group controlId="formPassword2">
+          <Form.Label>Re-Enter Password</Form.Label>
+          <Form.Control
+            type="password"
+            name="password2"
+            onChange={handleChange}
+            value={formData.password2}
+            placeholder="Re-enter password"
+            required
+          />
+        </Form.Group>
+
+        <Form.Group controlId="photoURL">
+          <Form.Label>Profile Pic URL</Form.Label>
+          <Form.Control
+            type="text"
+            name="photoURL"
+            onChange={handleChange}
+            value={formData.photoURL}
+            placeholder="Enter profile picture URL"
+          />
+        </Form.Group>
+
+        <Form.Group controlId="formControlSelectCity">
+          <Form.Label>Select your City</Form.Label>
+          <Form.Control
+            as="select"
+            name="city"
+            onChange={handleChange}
+            value={formData.city}
+            required
+          >
+            {cities.map((city, index) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <option key={index}>{city}</option>
```

Comment:
lets not have the index as a key here maybe it can be the city

Reply:
Taken a note in the bug bash file!